test/powershell/Modules/Microsoft.PowerShell.Utility/assets/docker/ClientCertificateCheck/Pages/_Layout.cshtml

```diff
@@ -0,0 +1 @@
+@RenderBody()
```

Comment:
Please check all files and add new line at EOF - some files have it and some files haven't it - we should use one pattern for all files.